easybuild/easyblocks/generic/bundle.py

```diff
@@ -162,7 +162,7 @@ def install_step(self):
 
     def make_module_extra(self):
         """Set extra stuff in module file, e.g. $EBROOT*, $EBVERSION*, etc."""
-        return super(Bundle, self).make_module_extra(altroot=self.altroot, altversion=self.altversion)
+        return EasyBlock.make_module_extra(self, altroot=self.altroot, altversion=self.altversion)
```

Comment:
there's a better way of fixing the issue you bumped into, see https://github.com/ocaisa/easybuild-easyblocks/pull/12